RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -752,6 +761,18 @@ bool MuonIdProducer::isGoodRPCMuon( const reco::Muon& muon )
   return ( muon.numberOfMatchedRPCLayers( reco::Muon::RPCHitAndTrackArbitration ) > minNumberOfMatches_ );
 }
 
+bool MuonIdProducer::isGEMMuon( const reco::Muon& muon )
+{
+  if(muon.track()->pt() < minPt_ || muon.track()->p() < minP_) return false;
+  return ( muon.numberOfMatches( reco::Muon::GEMSegmentAndTrackArbitration ) >= 1 );    
+}
+
+bool MuonIdProducer::isME0Muon( const reco::Muon& muon )
+{
+  if(muon.track()->pt() < minPt_ || muon.track()->p() < minP_) return false;
```

Comment:
minP of 3 GeV is probably going to be OK here.
Just removing the selection will bump up the CPU time.
An "easy" option is to leave it as is, but add a comment that this should be reviewed once we get to physics studies.